### PR TITLE
Update dx-graphics-hlsl-to-sample.md

### PR DESCRIPTION
**Pull request by briannagerads**

### CODE REVIEW

**Comment by PRMerger18** [comment]
@briannagerads : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: desktop-src/direct3dhlsl/dx-graphics-hlsl-to-sample.md [hunk 1]
```diff
@@ -157,6 +157,8 @@ VS_OUTPUT In;
 
 Texture sampling uses the texel position to look up a texel value. An offset can be applied to the position before lookup. The sampler state contains the sampling and filtering options. This method can be invoked within a pixel shader, but it is not supported in a vertex shader or a geometry shader.
 
+Use an offset only at an integer miplevel; otherwise, you may get results that do not translate well to hardware.
+
```
Comment (pow2clk) [inline desktop-src/direct3dhlsl/dx-graphics-hlsl-to-sample.md]: as we discussed, I think this should be under Applying texture coordinate offsets

File: desktop-src/direct3dhlsl/dx-graphics-hlsl-to-sample.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ Samples a texture.
 </tr>
 <tr class="even">
 <td><p><span id="Offset"></span><span id="offset"></span><span id="OFFSET"></span><em>Offset</em></p></td>
-<td><p>[in] An optional texture coordinate offset, which can be used for any texture-object type; the offset is applied to the location before sampling. Use an offset only at an integer miplevel; otherwise, you may get results that do not translate well to hardware. The argument type is dependent on the texture-object type. For more info, see Applying Integer Offsets.</p>
+<td><p>[in] An optional texture coordinate offset, which can be used for any texture-object type; the offset is applied to the location before sampling. The texture offsets need to be static. The argument type is dependent on the texture-object type. For more info, see <a href="https://docs.microsoft.com/en-us/windows/win32/direct3dhlsl/dx-graphics-hlsl-to-samplelevel">Applying Integer Offsets.</a></p>
```
Comment (pow2clk) [inline desktop-src/direct3dhlsl/dx-graphics-hlsl-to-sample.md]: I think each of these links should actually be to their own "Applying texture coordinate offsets" section.

**Comment by PRMerger20** [comment]
@briannagerads : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger7** [comment]
@briannagerads : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by stevewhims** [comment]
Thanks, @briannagerads. In fact, the original url is actually correct, and it works. We don't use fully-qualified urls where we can avoid them (as in this case), and in cases where we can't avoid them, we don't include the language tag.

I'll go ahead and close this one.

-Steve

**Pull request closed without merging**